pkg/paths/path.go

```diff
@@ -1,17 +1,19 @@
 package paths
 
 import (
-	"fmt"
-
 	"github.com/gobuffalo/buffalo"
+	"github.com/gomods/athens/pkg/errors"
 )
 
 // GetModule gets the module from the path of a ?go-get=1 request
 func GetModule(c buffalo.Context) (string, error) {
+	const op errors.Op = "paths.GetModule"
+
 	module := c.Param("module")
 	if module == "" {
-		return "", fmt.Errorf("module missing")
+		return "", errors.E(op, "module missing")
```

Comment:
I know it was there before, but maybe the string could be more descriptive. Something like "missing module parameter"?

Reply:
@robjloranger in my opinion we should even panic here. Because that's the Buffalo Handler misbehaving or we did not write the path regex correctly 😄 but will change the string text for now.